packages/itinerary-body/src/defaults/transit-leg-summary.tsx

```diff
@@ -32,7 +32,7 @@ export default function TransitLegSummary({
       <S.InvisibleAdditionalDetails>
         <FormattedMessage
           defaultMessage={defaultMessages["otpUi.TransitLegBody.expandDetails"]}
-          description="Screen reader text to expand stops"
+          description="Screen reader text added to expand steps"
```

Comment:
Why was this changed?

Reply:
I recall receiving some errors about this...let me dig into this further, but basically it was centered around there being different values for elements that were supposed to have the same thing. I updated to whichever entry was the most recent.